golem-worker-executor-base/src/durable_host/golem/mod.rs

```diff
@@ -132,7 +132,10 @@ impl<Ctx: WorkerCtx> golem::api::host::Host for DurableWorkerCtx<Ctx> {
         Ok(self
             .public_state
             .promise_service
-            .create(&self.worker_id.worker_id, self.state.oplog_idx)
+            .create(
+                &self.worker_id.worker_id,
+                self.state.current_oplog_index().await,
```

Comment:
Is this a logical change or refactor? :)  - trying to understand what's being done